app/src/test/java/org/mozilla/focus/utils/SupportUtilsTest.java

```diff
@@ -30,12 +32,20 @@ public void cleanup() {
      */
```

Comment:
Remove this comment, or state that there seems to be a Robolectric behavior change?